pallets/airdrop/src/lib.rs

```diff
@@ -234,6 +233,7 @@ pub mod pallet {
 	impl<T: Config> Pallet<T> {
 		/// Dispatchable to be called by server with privileged account
 		/// dispatch claim
+		#[pallet::call_index(0)]
```

Comment:
are we sure the call index generated prior is equal to what we are hardcoding now? we can make sure by looking into polkadot ui.

Reply:
thanks sabin for the tip and help on checking the call indices from the ui! i verified all call indices of the methods from 3 pallets and they match the values set in this pr.